net.bartkessels.simpleconvert.json

```diff
@@ -0,0 +1,38 @@
+{
+	"app-id": "net.bartkessels.SimpleConvert",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.9",
+	"sdk": "org.kde.Sdk",
+	"command": "simpleconvert",
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--socket=wayland",
+		"--filesystem=host"
```

Comment:
what's the problem with just using the portal, folders?

Reply:
The filesystem=host permission is needed for the ffmpeg process to write the input file(s) to the selected location.